repos/system_upgrade/common/actors/scaninstalledtargetkernelversion/tests/test_scaninstalledkernel_scaninstalledtargetkernelversion.py

```diff
@@ -7,10 +7,10 @@
 
 TARGET_KERNEL_VERSION = '1.2.3-4.el8.x86_64'
 TARGET_RT_KERNEL_VERSION = '1.2.3-4.rt56.7.el8.x86_64'
-TARGET_KERNEL = 'kernel-{}'.format(TARGET_KERNEL_VERSION)
-TARGET_RT_KERNEL = 'kernel-{}'.format(TARGET_RT_KERNEL_VERSION)
-OLD_KERNEL = 'kernel-0.1.2-3.el7.x86_64'
-OLD_RT_KERNEL = 'kernel-rt-0.1.2-3.rt4.5.el7.x86_64'
+TARGET_KERNEL = 'kernel-core-{}'.format(TARGET_KERNEL_VERSION)
+TARGET_RT_KERNEL = 'kernel-rt-core-{}'.format(TARGET_RT_KERNEL_VERSION)
+OLD_KERNEL = 'kernel-core-0.1.2-3.el7.x86_64'
+OLD_RT_KERNEL = 'kernel-rt-core-0.1.2-3.rt4.5.el7.x86_64'
```

Comment:
afaik there was no `kernel-core` on RHEL7 (`.el7` in the package name). IMHO, to make the change not misleading, the versions should be changed to `el7 -> el8` and `el8 -> el9` and the CurrentActorMocked instance should be modified to be from 8 to 9.

Alternatively, the old kernel packages don't have to be renamed at all, however, then the tests become trivial (and thus almost needless) as the rpm query underneath returns only one entry for the target kernel.

Reply:
If that is the case - the 7->8 e2e tests should fail, right?

Reply:
> afaik there was no `kernel-core` on RHEL7 (`.el7` in the package name). IMHO, to make the change not misleading, the versions should be changed to `el7 -> el8` and `el8 -> el9` and the CurrentActorMocked instance should be modified to so that the upgrade is from 8 to 9.
> 
> Alternatively, the old kernel packages don't have to be renamed at all, however, then the tests become trivial (and thus almost needless) as the rpm query underneath returns only one entry for the target kernel.

Good observation, going with the first option.